it/src/main/java/org/corfudb/universe/node/client/ClientParams.java

```diff
@@ -0,0 +1,42 @@
+package org.corfudb.universe.node.client;
+
+import lombok.Builder;
+import lombok.Builder.Default;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NonNull;
+import org.corfudb.universe.node.Node;
+import org.corfudb.universe.node.Node.NodeParams;
+import org.corfudb.universe.node.Node.NodeType;
+
+import java.time.Duration;
+
+@Builder
+@Getter
+@EqualsAndHashCode(exclude = {"numRetry", "timeout", "pollPeriod"})
+public
+class ClientParams implements NodeParams {
+    @Default
+    @NonNull
+    private final String name = "corfuClient";
+    @NonNull
+    private final NodeType nodeType = NodeType.CORFU_CLIENT;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)